lib/flipper/ui.rb

```diff
@@ -21,6 +21,8 @@ class << self
       # says "App" which points to this href. The href can be a path (ie: "/")
       # or full url ("https://app.example.com/").
       attr_accessor :application_breadcrumb_href
+      #Public: If set, then the add_feature button will be removed
+      attr_accessor :remove_add_feature_button
```

Comment:
I actually prefer to stay positive (enable_feature_creation or something) and default the positive to true than to have a negative (remove). I'll merge this and then tweak.